package.json

```diff
@@ -133,20 +133,20 @@
 		"css-loader": "3.2.0",
 		"cssnano": "4.1.10",
 		"deep-freeze": "0.0.1",
-		"enzyme": "3.9.0",
+		"enzyme": "3.11.0",
 		"eslint-plugin-eslint-comments": "3.1.2",
 		"eslint-plugin-import": "2.18.2",
-		"execa": "3.4.0",
+		"execa": "4.0.0",
```

Comment:
Changelog: https://github.com/sindresorhus/execa/releases

It shouldn't affect us. The changelog is perhaps confusing in how it mentions the addition of `stderr` and `stdout` as a breaking change. We have a few places we're using `stderr`, but my reading is that this impacts `error.message` specifically, and shouldn't impact our usage.